packages/p2p/src/components/modal-manager/modals/email-link-blocked-modal/email-link-blocked-modal.tsx

```diff
@@ -1,18 +1,21 @@
 import React from 'react';
-import PropTypes from 'prop-types';
 import { Icon, Modal, Text } from '@deriv/components';
-import { useModalManagerContext } from 'Components/modal-manager/modal-manager-context';
 import { Localize } from 'Components/i18next';
+import { useModalManagerContext } from 'Components/modal-manager/modal-manager-context';
+
+type TEmailLinkBlockedModalProps = {
+    email_link_blocked_modal_error_message: string;
+};
 
 const EmailLinkBlockedModal = ({
     // TODO: Uncomment when time is available in BE response
     // blocked_for_minutes,
     email_link_blocked_modal_error_message,
-}) => {
+}: TEmailLinkBlockedModalProps) => {
     const { hideModal, is_modal_open } = useModalManagerContext();
 
     return (
-        <Modal has_close_icon is_open={is_modal_open} renderTitle={() => <></>} toggleModal={hideModal} width='440px'>
+        <Modal is_open={is_modal_open} renderTitle={() => <></>} toggleModal={hideModal} width='440px'>
```

Comment:
Instead of passing `renderTitle` like this can we make it optional and checking that inside the modal component?

Reply:
@niloofar-deriv  i checked this. renderTitle is already an optional component and removing it will completely remove the header section which will hide the close icon as well, since we have this condition in modal.tsx => `{!is_risk_warning_visible && (header || title || rendered_title) && (`. it renders the header only if we have this condition satisfied. we can add a check for close icon as well, but it might break any design for the existing modals, since this is a shared component.

Reply:
you can put `title=' '`, i dont know if its better or not 😄

Reply:
wont it be same? both are optional parameters.

Reply:
Yeah, it just to show the header with close button without any title right? So the options are:
- renderTitle={() => <></>}
- title=' '
- header={<></>}

So, you can choose the simplest one

Reply:
title='' wont work because then the if condition will return false. it should be either renderTitle or header